File: app/src/main/java/com/ashwinrao/packup/util/UnitConversion.java
package com.ashwinrao.packup.util;

import android.content.Context;
import android.util.TypedValue;

public class UnitConversion {

    public static int dpToPx(Context context, float dp) {
        return (int) TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, context.getResources().getDisplayMetrics());
    }
}

File: app/src/main/java/com/ashwinrao/packup/util/UnitConversion.kt
package com.ashwinrao.packup.util

import android.content.Context
import android.util.TypedValue

object UnitConversion {

    @JvmStatic
    fun dpToPx(context: Context, dp: Float): Int = TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_DIP, dp, context.resources.displayMetrics).toInt()

}
